setup.py

```diff
@@ -429,5 +430,8 @@ def run(self):
           install_requires=[
             'mpmath>=%s' % mpmath_version,
             ],
+          extra_require={
+            'clang' : ["clang"],
```

Comment:
Currently we don't have any optional dependencies listed in setup.py. I would leave this out for now. If we want to add optional dependencies to setup.py, we should do so in a separate PR that adds all of them.

Reply:
ok, I'll leave that out for now.